__tests__/intergration.tests.js

```diff
@@ -54,3 +54,38 @@ describe("non existent endpoint", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET:200 returns the article with appropriate properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(typeof response.body.articles.article_id).toBe("number");
+        expect(typeof response.body.articles.votes).toBe("number");
+        expect(typeof response.body.articles.author).toBe("string");
+        expect(typeof response.body.articles.title).toBe("string");
+        expect(typeof response.body.articles.body).toBe("string");
+        expect(typeof response.body.articles.topic).toBe("string");
+        expect(typeof response.body.articles.created_at).toBe("string");
+        expect(typeof response.body.articles.article_img_url).toBe("string");
```

Comment:
I considered being more specific, but decided to keep it general in case a subsequent task edited the values, such as changing the number of votes etc - by keeping it a bit general, I was trying to avoid any potential errors in the future, without being less rigorous now. 
On thinking about that feedback though, I see now that I was being over cautious as the test data is reseeded every run, so that won't matter and could never be an issue, when working with test data.